### PR TITLE
[Improve] Add how to build on README

### PR DESCRIPTION
**Pull request by caicancai**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by caicancai** [comment]
@JNSimba Hello, Happy New Year. If you have time, can you help review this PR? Thank you.

File: README.md [hunk 1]
```diff
@@ -30,6 +30,15 @@ More information about compilation and usage, please visit [Spark Doris Connecto
 
 [Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0)
 
+## How to Build
+
+You need to copy customer_env.sh.tpl to customer_env.sh before build and you need to configure it before build.
+```shell
+git clone git@github.com:apache/doris-spark-connector.git
+cd doris-flink-connector/flink-doris-connector
```
Comment (JNSimba) [inline README.md:38]: may be spark？

Comment (caicancai) [reply README.md]: done，thanks

**Review by JNSimba — APPROVED** [review]
LGTM

**Pull request merged**